physionet-django/console/views.py

```diff
@@ -126,16 +126,38 @@ def submitted_projects(request):
 
     assign_editor_form = forms.AssignEditorForm()
 
+    # Time to check if the reminder email can be sent
+    yesterday = timezone.now() + timezone.timedelta(days=-1)
```

Comment:
doesn't really matter, but why not `timezone.now() - timezone.timedelta(days=1)`? seems clearer to me.